components/elevation/src/vwc-elevation.scss

```diff
@@ -1,8 +1,6 @@
 @use 'sass:list';
 
 $dps: 0 2 4 8 12 16 24;
-$vvd-elevation-background-color: --vvd-elevation-background-color;
```

Comment:
These constants prevent typos that may occur by authoring "magic strings".
Even if used in a single page, renaming a variable may leave a reference outdated. While constants will break the compilation early

Reply:
I removed that line anyway :)

Reply:
right, don't. it helps ensure we use the right css variable. rule of thumb